pkg/services/featuremgmt/registry.go

```diff
@@ -90,7 +90,7 @@ var (
 			Name:        "logRequestsInstrumentedAsUnknown",
 			Description: "Logs the path for requests that are instrumented as unknown",
 			Stage:       FeatureStageExperimental,
-			Owner:       hostedGrafanaTeam,
+			Owner:       grafanaBackendServicesSquad,
```

Comment:
@hairyhenderson  / @rhorvath ptal at this and other flags

Reply:
the original agreement was that logging would be a group-wide shared responsibility - can we change this to `grafanaBackendGroup` instead for now?